packages/sit-onyx/src/components/OnyxInput/OnyxInput.vue

```diff
@@ -77,9 +82,21 @@ const shouldShowCounter = computed(() => props.withCounter && props.maxlength);
     <label>
       <div
         v-if="!props.hideLabel"
-        :class="['onyx-input__label', 'onyx-text--small', requiredMarkerClass]"
+        class="onyx-input__label onyx-text--small"
+        :class="[!props.required ? requiredMarkerClass : undefined]"
       >
-        <div class="onyx-truncation-ellipsis">{{ props.label }}</div>
+        <div class="onyx-input__header">
+          <span class="onyx-truncation-ellipsis">
+            {{ props.label }}
+          </span>
+          <span :class="[props.required ? requiredMarkerClass : undefined]"></span>
+          <OnyxTooltip v-if="props.labelTooltip" :text="props.labelTooltip">
```

Comment:
replace this with `OnyxInfoTooltip` (also below for the message)

Reply:
Apparently I only did this for textarea :)